HelpMyStreetFE/HelpMyStreetFE/Services/AddressService.cs

```diff
@@ -62,6 +67,7 @@ public class AddressService : BaseHttpService, IAddressService
             _memDistCache_LocationDetailsList = memDistCache_LocationDetailsList;
             _memDistCache_LocationDistanceList = memDistCache_LocationDistanceList;
             _memDistCache_PostcodeDistances = memDistCache_PostcodeDistances;
+            _httpContextAccessor = httpContextAccessor;
```

Comment:
Can we now lose the HttpContectAccessor from here, and potentially some of the caches too?